<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MadLibs_Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{
background:orange;
}
QLabel
{
	font-size:50px;
	font-weight:bold;
	font-style:italic;
}
QPushButton{
	color:blue;
	font-weight:bold;
	font-family:'rockwell';
	border:2px solid red;
	background:white;
	border-radius:10px;



}

#madlib1btn:hover,#madlib2btn:hover,#madlib3btn:hover{
	background:blue;
  	color:white;
}</string>
  </property>
  <widget class="QLabel" name="welcome">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </widget>
  <widget class="QLabel" name="to">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>MadLibs</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue</string>
   </property>
   <property name="text">
    <string>Generator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="madlib1btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Madlib 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="madlib2btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Madlib 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="madlib3btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Madlib 3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
